skyvern/forge/sdk/db/client.py

```diff
@@ -2250,3 +2252,96 @@ async def get_workflow_run_blocks(
                 convert_to_workflow_run_block(workflow_run_block, task=tasks_dict.get(workflow_run_block.task_id))
                 for workflow_run_block in workflow_run_blocks
             ]
+
+    async def get_active_persistent_browser_sessions(self, organization_id: str) -> List[PersistentBrowserSession]:
+        """Get all active persistent browser sessions for an organization."""
+        async with self.Session() as session:
+            result = await session.execute(
+                select(PersistentBrowserSessionModel).where(
+                    PersistentBrowserSessionModel.organization_id == organization_id,
+                    PersistentBrowserSessionModel.deleted_at.is_(None),
+                )
+            )
+            sessions = result.scalars().all()
+            return [PersistentBrowserSession.model_validate(session) for session in sessions]
+
+    async def get_persistent_browser_session(
+        self, session_id: str, organization_id: str
+    ) -> Optional[PersistentBrowserSessionModel]:
+        """Get a specific persistent browser session."""
+        async with self.Session() as session:
+            result = await session.execute(
+                select(PersistentBrowserSessionModel).where(
+                    PersistentBrowserSessionModel.persistent_browser_session_id == session_id,
+                    PersistentBrowserSessionModel.organization_id == organization_id,
+                )
+            )
+            return result.scalar_one_or_none()
+
+    async def create_persistent_browser_session(
+        self,
+        organization_id: str,
+        runnable_type: str | None = None,
+        runnable_id: str | None = None,
+    ) -> PersistentBrowserSessionModel:
+        """Create a new persistent browser session."""
+        async with self.Session() as session:
+            db_session = PersistentBrowserSessionModel(
+                organization_id=organization_id,
+                runnable_type=runnable_type,
+                runnable_id=runnable_id,
+            )
+            print(db_session)
+            session.add(db_session)
+            await session.commit()
+            await session.refresh(db_session)
+            return PersistentBrowserSession.model_validate(db_session)
+
+    async def mark_persistent_browser_session_deleted(self, session_id: str, organization_id: str) -> None:
+        """Mark a persistent browser session as deleted."""
+        LOG.info(
```

Comment:
Another convention: we usually delegate such a log to files that are not in this db client leve.